### PR TITLE
`mapping_set_id` corrected

### PR DESCRIPTION
**Pull request by hrshdhgd**
Addresses #5846

### CODE REVIEW

File: src/ontology/metadata/mondo.sssom.config.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ global_metadata:
   subject_source: http://purl.obolibrary.org/obo/mondo.owl
   license: "https://creativecommons.org/publicdomain/zero/1.0/"
   mapping_date: "2017-09-20"
-  mapping_set_id: "GLOBAL" # Apparently this is the one that matters.
+  mapping_set_id: "http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv" # Apparently this is the one that matters.
```
Comment (matentzn) [inline src/ontology/metadata/mondo.sssom.config.yml:100]: Is this needed when `sssom annotate`?

**Review by matentzn — CHANGES_REQUESTED** [review]
- Many `object_label` columns have disappeared from the split file
- Can we add `object_label` to the main mondo.sssom.tsv file?

File: src/ontology/metadata/mondo.sssom.config.yml (hunk 1 above)
Comment (hrshdhgd) [reply src/ontology/metadata/mondo.sssom.config.yml:100]: not really. I'll change it.

**Comment by hrshdhgd** [comment]
I think the latest run fixed the missing `object_label` issue. I had to run `make mondo-ingest.owl` and `semsql make tmp/mondo-ingest.db -P config/prefixes.csv` again before `make mappings`.

**Review by matentzn — COMMENTED** [review]
This looks great! Almost there! Can you 

1. Make sure that we don't have to run anything manually? Maybe update https://github.com/monarch-initiative/mondo/blob/550263a2d62d6a468e2883421de654030c42c5af/src/ontology/mondo.Makefile#L641?
2. Deal with https://github.com/monarch-initiative/mondo/issues/4751? I still see sorting issues in your diff, see for example `src/ontology/mappings/mondo_exactmatch_mesh.sssom.tsv`.

Remember this pipeline is updated with every release, so we need to make sure that when Sabrina runs the release, the files get updated in the correct way.

**Review by matentzn — APPROVED** [review]

**Pull request merged**